server/szurubooru/migrations/functions.py

```diff
@@ -0,0 +1,68 @@
+from alembic_utils.pg_function import PGFunction
```

Comment:
Is writing a SQL function really necessary instead of doing it in SQLAlchemy? This is something that can get very hard to maintain in the future